composer.json

```diff
@@ -41,7 +41,15 @@
         "drupal/relaxed": "~1",
         "drupal/multiversion": "~1",
         "drupal/workspace": "~1",
-        "drupal/adobeanalytics": "dev-8.x-1.x"
+        "drupal/adobeanalytics": "dev-8.x-1.x",
```

Comment:
Have opted to fix to a version as we need "control" over the modules we are using.